.github/actions/merge-base-branch/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Merge from Base Branch if Action Running on a PR'
+description: 'Merge from the base branch the PR is being merged into.'
+runs:
+  using: 'composite'
+  steps:
+    - id: merge
+      name: Merge
+      run: |
+        if [[ ${{ github.event_name}} != "pull_request" ]]
+        then
+          echo "Skipping merge because not triggered by a PR event."
+          exit 0
+        fi
+        echo "Base Repository: ${{ github.repository }}"
+        echo "Base Branch: ${{ github.base_ref }}"
+        echo "Head Repository: ${{ github.event.pull_request.head.repo.full_name }}"
+        echo "Head Branch: $GITHUB_HEAD_REF"
+        git config --global user.email "admin@oppia.org"
+        git config --global user.name "Oppia Merge Base Branch GitHub Action"
```

Comment:
Do we need this step, if yes should we use dummy email/username?

Reply:
Yes we need this. Without it, `git` will refuse to make the commit. This commit exists only for the duration of the CI check since we don't push it anywhere. Therefore, I don't think it really matters what the email and username are. I made the user name descriptive so that anyone debugging (e.g. by adding a step to execute `git log`) would know where it came from

Reply:
Leaving it to admin to decide, @seanlip Do you think it's fine to use this email?

Reply:
No, I don't think so. I suggest using a made-up email address that explains what it is for, e.g. tempaccount.for@merge.check or similar, since these commits are not being made by the Oppia admin.

Also, it would be better not to need these statements at all if possible. What error message exactly do you get when this is not set?

Reply:
This is required because merging creates a merge commit, and creating a commit requires a name and email. If we don't do this configuration, we'll get an error like this:

```
*** Please tell me who you are.

Run

git config --global user.email "you@example.com"

git config --global user.name "Your Name"

to set your account's default identity.

Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got '...')
```

Reply:
I've changed the email to use `example.com`